### ISSUE TITLE
OrderModifiers are not called anymore

### ISSUE THREAD

**Issue by aurelien-cd** [lunarphp/lunar#1095]
- Lunar version: 0.2.0
- Laravel Version: 9.52.8
- PHP Version: 8.2.4
- Database Driver & Version: Mysql 8

Since 0.2.0, the order modifiers declared are not called anymore, even if the OrderModifiers class is still present, and the docs are still mentioning them.

Specifically, these lines have disappeared in `Lunar\Actions\Carts\CreateOrder` :
```php
$pipeline = app(Pipeline::class)
   ->send($cart)
   ->through(
      $this->getModifiers()->toArray()
   );

$cart = $pipeline->via('creating')->thenReturn();
```
and
```php
private function getModifiers()
{
   return app(OrderModifiers::class)->getModifiers();
}
```

It seems that since 0.2.0 cart modifiers have been removed in favor of the new configurable cart pipelines. But i can't see any replacement solution in place for orders.

BTW, the `Lunar\Base\CartModifiers` is still present in codebase, even if not used anymore.

**Comment by glennjacobs** [comment]
@aurelien-cd It appears that Order Modifiers were lost when the Cart refactoring was done.

We are going to re-introduce this functionality as Pipelines so it takes the same approach as the Cart. Open to contributions if anyone has time.

You should expect this for the 0.4 release at the start of July.

**Comment by alecritson** [comment]
This will be resolved in #1097
